poetry.toml

```diff
@@ -0,0 +1,2 @@
+[virtualenvs]
+in-project = true
```

Comment:
We should probably add a `create = true` line here (although it's highly unlikely that we'll have someone who's changed this from the default of `true`)